buildscripts/templates/SemanticAttributes.java.j2

```diff
@@ -28,8 +28,8 @@
     {{type | to_camelcase(False)}}Key
   {%- endif -%}
 {%- endmacro %}
-{%- macro print_value(type, value) -%}
-  {{ "\"" if type == "String"}}{{value}}{{ "\"" if type == "String"}}
+{%- macro stable_class_ref(const_name) -%}
+{{stablePkg}}.{{file_name[:-5].replace("Incubating", "")}}#{{const_name}}
```

Comment:
`file_name` is  `{root_namespace}IncubatingAttributes.java`

this is ugly and relies on certain way to name files, so might need to think about better way in the future.

For now we don't really need to generate any stable attributes in the incubating artifact.

Reply:
> For now we don't really need to generate any stable attributes in the incubating artifact.

Are you suggesting we punt on this whole PR until some later point?